admin/tests/integration/components/sessions/jury-comment_test.js

```diff
@@ -364,3 +363,7 @@ async function _isNotInEditMode() {
     throw new Error('Component should not be in edit mode, but it is.');
   }
 }
+
+function _formatDate(date) {
+  return moment(date).format('DD/MM/YYYY à HH:mm');
+}
```

Comment:
```suggestion
function _formatDate(dateString) {
  return moment(new Date(dateString)).format('DD/MM/YYYY à HH:mm');
}
```

Une série de suggestions pour mettre des valeurs explicites dans le `then`.

Reply:
done